src/panels/config/devices/device-detail/ha-device-info-card.ts

```diff
@@ -106,13 +107,31 @@ export class HaDeviceCard extends LitElement {
                 </div>
               `
             : ""}
+          ${this._getAddresses().map(
+            (connection) => html`
+              <div class="extra-info">
+                ${titleCase(connection[0])}: ${connection[1].toUpperCase()}
```

Comment:
This will cause it to be `Mac` but it should be `MAC` (as it is an abbreviation).